src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-card flat id="upload-memorandum">
+
+    <!-- Summary Section -->
+    <div id="upload-memorandum-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="upload-memorandum-summary-header" >
+        <v-icon color="#38598A">mdi-text-box-multiple</v-icon>
+        <label class="upload-memorandum-title"><strong>Memorandum</strong></label>
+      </div>
+
+      <!-- Summary error message -->
+      <div v-if="!valid" class="upload-memorandum-error-message">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text"> This step is not complete. </span>
+          <router-link :to="{ path: `/${RouteNames.CREATE_MEMORANDUM}`, query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <v-card-title>Memorandum Placeholder component</v-card-title>
+  </v-card>
+</template>
+<script lang="ts">
+// Libraries
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { State } from 'vuex-class'
+
+// Enums
+import { RouteNames } from '@/enums'
+
+@Component({})
+export default class UploadMemorandum extends Vue {
+  // Global state
+  @State(state => state.stateModel.createRulesStep.valid)
+  readonly valid!: boolean
+
+  @Prop({ default: false })
+  private isSummary: boolean
+
+  // Entity Enum
+  readonly RouteNames = RouteNames
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+.upload-memorandum-summary-header {
+  display: flex;
+  background-color: $BCgovBlue5O;
+  padding: 1.25rem;
+
+  .upload-memorandum-title {
+    padding-left: 0.5rem;
+  }
+}
+
+.upload-memorandum-error-message {
+  padding-top: 1.25rem;
+  padding-left: 1.25rem;
```

Comment:
I know much other code in this project specifies layout in the "style" block, but IMHO these should use Vuetify helper classes in the HTML. (Exception: if we implement standard classes for different things in a common style file.)
Don't rewrite this but I'd prefer to see inline layout classes in future code.

Reply:
Ok fair enough, i see your point. 
My thought process was, we needed class for the `color` so i put the stylings in a single place. 
Curious though, i'm not sure if the helper classes are in rem, pretty sure they are set pixels no? Something like 4px per digit?

Reply:
TBH I'm not hard and fast on this even in my own code (and some classes are only created at run-time so you have to use the style block) (and sometimes you want to do something for only the first, or last, element in a list so you have to use the style block).

Even with the need for a colour style (though you could use a base class to set fg or bg colour ;) ), I try to have layout styles in the HTML.

I try to do what looks clean or feels right. I presume you've done the same so just leave it :)

Reply:
^^ in other projects we have base classes like "red-text", etc

Reply:
^^ but I'm not asking you to change it! :)